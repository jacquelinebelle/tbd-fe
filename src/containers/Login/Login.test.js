import React from 'react';
import { shallow } from 'enzyme';
import { Login } from './index';

describe("Login", () => {
  let wrapper;

  beforeEach(()=> {
    wrapper = shallow(
      <Login />
    )
  }) 
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

});
