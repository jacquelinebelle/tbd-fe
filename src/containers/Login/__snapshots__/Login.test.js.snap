// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should match the snapshot 1`] = `ShallowWrapper {}`;
